web/src/styles/globals.css

```
@@ -38,6 +46,26 @@ body {
 @layer base {
   * {
     @apply border-[var(--gantry-border)];
+    scrollbar-color: var(--gantry-bg-tertiary) var(--gantry-bg-secondary);
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)